src/main/java/uk/ac/ebi/subs/validator/filereference/messaging/FileReferenceRoutingKeys.java

```diff
@@ -0,0 +1,12 @@
+package uk.ac.ebi.subs.validator.filereference.messaging;
```

Comment:
This should be moved to `uk.ac.ebi.subs.validator.messaging` where all the messaging configuration classes are for the rest of the components.